policy/goals/types.go

```diff
@@ -40,13 +40,17 @@ type (
 		Digest       string `edn:"docker.image/digest"`
 	}
 
+	RetractionEntity struct {
```

Comment:
This could probably be moved to somewhere more common since it isn't a policy specific concept. But fine for now.